README.md

```diff
@@ -112,6 +112,25 @@ These configurations should all be runnable for local testing, except for `compl
 You will need access to the vault to run the `githubauth` configuration, which requires secrets for the github oauth2 client app
 details.
 
-### Notes
+## Running locally
 
-If developing in **basic auth** mode, a super admin user can be created by running `./scripts/create-super-user.sh` after the app is running.
+For example, to bring up the `basicauth` configuration, you can run:
+
+```console
+hatch env run -- packit start --pull config/basicauth
+./scripts/create-super-user
+```
```

Comment:
It happens on the other branches too, but I reliably get competition for use of my proxy ports 80 and 443 (no idea what's using them), so I always have to update those in the config to be some unused ports.

If that's a common problem it could be worth mentioning in the README.